artacle-swagger.yaml

```diff
@@ -825,6 +827,641 @@ paths:
         '500':
           $ref: "#/components/responses/InternalServerError"
 
+  #  Tokens
+  /collections/{cid}/tokens/{tid}/valuation/v1:
+    get:
+      tags:
+        - Tokens
+      summary: Corporate PRO only. Get particular token valuation information. All volumes by default are provided in the native collection currency (i.e. ETH for Ethereum)
```

Comment:
There are no volumes for tokens, I suggest to replace the word with 'prices' or something